cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/ConfigurationsActionButtons/index.js

```diff
@@ -31,17 +31,36 @@ const mapStateToProps = (state, ownProps) => {
     validToSave: state.validToSave,
     pipelineEdited: state.pipelineEdited,
     updatingPipeline: ownProps.updatingPipeline,
+    updatingPipelineAndAction: ownProps.updatingPipelineAndAction,
     saveAndClose: ownProps.saveAndClose,
+    saveAndAction: ownProps.saveAndAction,
     activeTab: ownProps.activeTab,
+    actionLabel: ownProps.actionLabel
   };
 };
 
-const ConfigActionButtons = ({runtimeArgs, validToSave, pipelineEdited, updatingPipeline, saveAndClose, activeTab}) => {
+const ConfigActionButtons = ({runtimeArgs, validToSave, pipelineEdited, updatingPipeline, updatingPipelineAndAction, saveAndClose, saveAndAction, activeTab, actionLabel}) => {
```

Comment:
Sorry for reason the names seems a little ambiguous to me. `updatingPipelineAndAction`, `saveAndClose` & `saveAndAction`.  Can we make them a little bit more easier?

Reply:
Have changed all the `Action` references to `SaveOrSchedule`, and `Close` to `CloseModeless`. Hope that clears it up a little bit.